### ISSUE TITLE
Causing Item renderer crashes

### ISSUE THREAD

**Issue by UltraBlackLinux** [xMrVizzy/Skyblocker#3]
```
[20:45:56] [main/FATAL]: Unreported exception thrown!
java.lang.NullPointerException: null
	at Not Enough Crashes deobfuscated stack trace.(1.16.4+build.7) ~[?:?]
	at net.minecraft.client.render.item.ItemRenderer.handler$zjl000$renderItemBar(ItemRenderer:1532) ~[?:?]
	at net.minecraft.client.render.item.ItemRenderer.renderGuiItemOverlay(ItemRenderer) ~[?:?]
	at me.shedaniel.rei.impl.ItemEntryStack.optimisedRenderOverlay(ItemEntryStack.java:373) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at me.shedaniel.rei.gui.widget.EntryListWidget.renderEntries(EntryListWidget.java:235) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at me.shedaniel.rei.gui.widget.EntryListWidget.lambda$render$5(EntryListWidget.java:307) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at java.util.HashMap.forEach(HashMap.java:1289) ~[?:1.8.0_275]
	at me.shedaniel.rei.gui.widget.EntryListWidget.render(EntryListWidget.java:306) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at me.shedaniel.rei.gui.ContainerScreenOverlay.renderWidgets(ContainerScreenOverlay.java:648) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at me.shedaniel.rei.gui.ContainerScreenOverlay.render(ContainerScreenOverlay.java:555) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at me.shedaniel.rei.RoughlyEnoughItemsCore.lambda$registerClothEvents$15(RoughlyEnoughItemsCore.java:539) ~[67ef00c5-a894-4c62-a3f0-6f633d89ccbb.jar:?]
	at net.minecraft.client.gui.screen.Screen.handler$bgd000$onPostDraw(Screen:3112) ~[?:?]
	at net.minecraft.client.gui.screen.Screen.render(Screen:83) ~[?:?]
	at net.minecraft.client.gui.screen.ingame.HandledScreen.render(HandledScreen:106) ~[?:?]
	at net.minecraft.client.gui.screen.ingame.AbstractInventoryScreen.render(AbstractInventoryScreen:43) ~[?:?]
	at net.minecraft.client.gui.screen.ingame.InventoryScreen.render(InventoryScreen:92) ~[?:?]
	at net.minecraft.client.render.GameRenderer.render(GameRenderer:616) ~[?:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1048) ~[?:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:681) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:215) [intermediary-minecraft-1.16.4-client.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_275]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_275]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_275]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_275]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.1.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139) [fabric-loader-0.11.1.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27) [fabric-loader-0.11.1.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_275]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_275]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_275]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_275]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
```

```
java.lang.NullPointerException
    at Not Enough Crashes deobfuscated stack trace.(1.16.4+build.7)
    at net.minecraft.client.render.item.ItemRenderer.handler$zjl000$renderItemBar(ItemRenderer:1532)
    at net.minecraft.client.render.item.ItemRenderer.renderGuiItemOverlay(ItemRenderer)
    at me.shedaniel.rei.impl.ItemEntryStack.optimisedRenderOverlay(ItemEntryStack.java:373)
    at me.shedaniel.rei.gui.widget.EntryListWidget.renderEntries(EntryListWidget.java:235)
    at me.shedaniel.rei.gui.widget.EntryListWidget.lambda$render$5(EntryListWidget.java:307)
    at java.util.HashMap.forEach(HashMap.java:1289)
    at me.shedaniel.rei.gui.widget.EntryListWidget.render(EntryListWidget.java:306)
    at me.shedaniel.rei.gui.ContainerScreenOverlay.renderWidgets(ContainerScreenOverlay.java:648)
    at me.shedaniel.rei.gui.ContainerScreenOverlay.render(ContainerScreenOverlay.java:555)
    at me.shedaniel.rei.RoughlyEnoughItemsCore.lambda$registerClothEvents$15(RoughlyEnoughItemsCore.java:539)
    at net.minecraft.client.gui.screen.Screen.handler$bgd000$onPostDraw(Screen:3112)
    at net.minecraft.client.gui.screen.Screen.render(Screen:83)
    at net.minecraft.client.gui.screen.ingame.HandledScreen.render(HandledScreen:106)
    at net.minecraft.client.gui.screen.ingame.AbstractInventoryScreen.render(AbstractInventoryScreen:43)
    at net.minecraft.client.gui.screen.ingame.InventoryScreen.render(InventoryScreen:92)
    at net.minecraft.client.render.GameRenderer.render(GameRenderer:616)
    at net.minecraft.client.MinecraftClient.render(MinecraftClient:1048)
    at net.minecraft.client.MinecraftClient.run(MinecraftClient:681)
    at net.minecraft.client.main.Main.main(Main:215)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
    at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
    at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
    at org.multimc.EntryPoint.listen(EntryPoint.java:143)
    at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by xMrVizzy** [comment]
Okay will look it today

**Comment by xMrVizzy** [comment]
it looks like REI problem

**Comment by xMrVizzy** [comment]
because in this log zero classes related to me.xmrvizzy.skyblocker package!

**Comment by UltraBlackLinux** [comment]
> because in this log zero classes related to me.xmrvizzy.skyblocker package!

Yeah, some mod also tells me it's caused by the REI Runtime, but this only occurs when I have your mod enabled :/

**Comment by xMrVizzy** [comment]
Fixed in v1.0.4 - https://modrinth.com/mod/skyblocker/version/okrToUiZ

**Comment by UltraBlackLinux** [comment]
@xMrVizzy 
Also Modrinth is always telling me, the page is not found :thinking:
